easybuild/easyconfigs/b/Bio-SearchIO-hmmer/Bio-SearchIO-hmmer-1.7.3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PerlModule'
+
+name = 'Bio-SearchIO-hmmer'
+version = '1.7.3'
+
+homepage = 'https://metacpan.org/pod/Bio::SearchIO::hmmer3'
+description = """Code to parse output from hmmsearch, hmmscan, phmmer and nhmmer, compatible
+with both version 2 and version 3 of the HMMER package from http://hmmer.org."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['686152f8ce7c611d27ee35ac002ecc309f6270e289a482993796a23bb5388246']
+
+dependencies = [
+    ('Perl', '5.32.0'),
+    ('BioPerl', '1.7.8'),
+]
+
+options = {'modulename': 'Bio::SearchIO::hmmer3'}
+
+sanity_check_paths = {
+    'files': ['bin/bp_%s.pl' % x for x in ['hmmer_to_table', 'parse_hmmsearch']],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/x86_64-linux-thread-multi/auto'],
```

Comment:
Hm, are we building this? Might be best to leave out that platform specific subdirectory

Reply:
done